packages/dom/src/dom.js

```diff
@@ -257,14 +257,16 @@ export function computeCaretRect() {
  *
  * @param {Element} container Focusable element.
  * @param {boolean} isReverse True for end, false for start.
+ * @param {Object} focusOptions options that get passed to the focus() call.
```

Comment:
The alignment of these params isn't right. Here's an example of the preferred alignment:
https://github.com/WordPress/gutenberg/blob/master/packages/rich-text/src/get-line-index.js